_posts/2018-03-15-release-3-3-0.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Release 3.3.0"
+date: 2018-03-16 10:00:00
+author: "@mikaelrandy"
+categories: release
+---
+
+We are proud to announce the availability of atoum [3.3.0][milestone]!
+
+## Stats
+
+* 43 [commits][commits],
+* 129 files changed,
+* 7 contributors,
+* 13 [PR closed][changelog].
+
+## What's new
+
+As we work on the next [major release], this release focus on stabilization and bug fixes, but there is also some new features.
+
+#### `string::notMatches` assertion
+The new `notMatches` assertion on `string` asserter allow to test that a string do not match a regexp.
+
+#### `@os` annotation
+If you want to write OS specific tests, atoum now provide an `@os` assertion to check OS before launching test
+
+Here is an example : 
+```php
+namespace tests\units
+{
+    /**
+     * @os !darwin
+     */
+    class stdClass extends \mageekguy\atoum\test
+    {
+        /**
+         * @os linux
+         */
+        public function testOnlyOnLinux()
+        {
+
+        }
+
+        /**
+         * @os !linux
+         */
+        public function testEverywhereButLinuxAndDarwin()
+        {
+
+        }
+    }
+}
+```
+
+#### Dot report
+If you need a condensed output of atoum execution, we had a new Dot report who only display test execution progression, without any other details :
+
+```$ bin/atoum -udr -f tests/units/classes/test.php
+............................................................ [60/65]
+.....                                                        [65/65]
+
+Success (1 test, 65/65 methods, 0 void method, 0 skipped method, 872 assertions)!
+```
+
```

Comment:
I would add a link to the documentation, saying that the documentation has already been updated ;)

Reply:
I think the documentation should be always up to date, and it should not be necessary to specify this point.